docs/contributing/local-development.md

```diff
@@ -10,14 +11,33 @@
 
 Tilt offers a simple way of creating a local development environment.
 
-## Create a kind cluster
+## Create local k8s cluster
 
-Create a kind cluster with a local registry:
+```bash
+make deploy-dev
+```
+
+Open the `tilt` browser UI by pressing the "space" key when prompted. Note that `tilt` sets up a directory watcher and will automatically redeploy any code changes!
+
+You can check the "temporal-operator-controller-manager" Pod status with `kubectl get pods -n temporal-system -w`.
+
+## Apply Manifests
+
+Once the local cluster is created, start applying your desired manifests and let the temporal operator handle reconciliation:
 
 ```bash
-make dev-cluster
+# example
+make artifacts
+kubectl apply -f out/release/artifacts/temporal-operator.crds.yaml
```

Comment:
CRDs and controller are already applied by `tilt`

Reply:
wasn't aware. Ok so I removed application of CRDs and operator:
```yaml
kubectl apply -f out/release/artifacts/temporal-operator.crds.yaml # REMOVED
kubectl apply -f out/release/artifacts/temporal-operator.yaml # REMOVED
```